packages/react-form/src/hooks/test/form-with-dynamic-list.test.tsx

```diff
@@ -142,4 +143,111 @@ describe('useForm with dynamic list', () => {
       expect(isDirty(wrapper)).toBe(false);
     });
   });
+
+  describe('makeCleanAfterSubmit', () => {
+    it('does not set dirty to false after successful submit by default', async () => {
```

Comment:
This tests seems to be the same as "does not set dirty to false after successful submit if makeCleanAfterSubmit is false", I think we can remove it. We don't care about testing the default values of `FormWithDynamicVariantList`, this is just a test component.